lib/tests/testscripts/scripts/reporting/multisystem-evaluate-measure.xml

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TestScript xmlns="http://hl7.org/fhir">
+  <id value="multisystem-evaluate-measure-xml"/>
+
+  <url value="http://example.com"/>
+  <name value="Individual Patient Multisystem TestScript for Evaluating Measures--XML"/>
+  <status value="draft"/>
+  <date value="2019-02-11"/>
+  <publisher value="MITRE"/>
+  <contact>
+    <name value="Matthew Gramigna"/>
+    <telecom>
+      <system value="email"/>
+      <value value="mgramigna@mitre.org"/>
+      <use value="work"/>
+    </telecom>
+  </contact>
+  <description value="Tests using XML format to execute the $evauate-measure operation. The destination server must support the $evaluate-measure operation on the Measure resource."/>
+  <copyright value="N/A"/>
+
+  <fixture id="example-measure">
+    <resource>
+      <reference value="./_reference/resources/Measure/measure-col.json"/>
+    </resource>
+  </fixture>
+
+  <variable>
+    <name value="measureID"/>
+    <defaultValue value="measure-col"/>
+  </variable>
+  <variable>
+    <name value="KnownPatientResourceId"/>
+    <defaultValue value="Patient-6516"/>
```

Comment:
I feel like a TestScript shouldn't depend on existing resources on the server, and instead should use create or update to ensure they're there. This would go for this patient record and any related data relevant to calculation measure-col measure.

Reply:
So would that essentially mean that this would become a submit data testscript that operates on two servers? It's fine if that's the case, I just want to confirm.

I agree that it doesn't make as much intuitive sense to have a testscript be dependent on data existing on the server so it's a worthwhile change regardless.

Reply:
Yeah it would be pretty similar to the `submit-data` TestScript, although I'd just use `update` and/or `create` endpoints (you might prefer update because you can control the ID, i.e. if you call update on a resource that doesn't exist, it will just create it at the ID in the URL path) rather than `$submit-data` in order to isolate `$evaluate-measure` behavior as much as possible, and because a server could theoretically have implemented `$evaluate-measure` but not `$submit-data`.

Reply:
I added a setup section that creates/updates the resources, similar to how it's done in submit data